### PR TITLE
Fix paths for modules views (phpvms#1546)

### PR DESCRIPTION
**Pull request by arthurpar06**
As I was talking about it the other day I think everything works, I only tested it with my laravel 10 branch of phpvms so maybe it doesn't work on laravel 9, it should be tested, if necessary we can wait for nabeel to merge the pr laravel 10 to merge this pr

### CODE REVIEW

**Comment by FatihKoz** [comment]
Will try to test this with laravel9 on local and live servers :)

With and without duplicated module blades.

**Comment by FatihKoz** [comment]
Technically, it is not working with latest dev, so not mergeable at the moment

1. Generates exception 'cause of non defined variable ($sourcePath), easy to solve
2. Functionality is lost, with latest dev module only uses its own blades, it does not load customized blades from user theme

Basically a module service provider should

- Serve its own blades (original as designed by the author)
phpvms_root/modules/**DisposableBasic**/Resources/views

- Customized blades (as the user wants to change, update them)
phpvms_root/resources/views/layouts/{**MyTheme**}/modules/**DisposableBasic**

If we can make this work with current/latest dev before laravel10 we can merge, if not we should wait :)

File: Providers/DB_ServiceProvider.php [hunk 1]
```diff
@@ -157,9 +157,12 @@ public function registerViews()
 
         $this->publishes([$sourcePath => $viewPath,], 'views');
 
-        $this->loadViewsFrom(array_merge(array_map(function ($path) {
-            return $path . '/modules/DisposableBasic';
-        }, \Config::get('view.paths')), [$sourcePath]), 'DBasic');
+        $this->loadViewsFrom(array_map(function ($path) {
```
Comment (FatihKoz) [inline Providers/DB_ServiceProvider.php:160]:
```suggestion
        $this->loadViewsFrom(array_map(function ($path) use ($sourcePath) {
```
Will solve the undefined variable exception

**Comment by FatihKoz** [comment]
Closing this, as looks like current implementation works fine with laravel 10/beta.5 and latest dev builds without errors and module still allows admins/va owners to have their customized blades under their theme folders.

**Comment by arthurpar06** [comment]
No problem, I'll watch again somethime. The idea was to fix the `artisan view:cache` command which didn't work because some paths were not existing (like /phpvms_root/resources/views/modules/DisposableBasic). However I don't remember that this prevented you from having your own blade files in your theme folder but if it did it's clear that it wasn't the right fix

**Comment by FatihKoz** [comment]
Yeah unfortunately it was breaking the customization ability, we can look at the logic later to remove non necessary paths and keep it simple with two folders (module + theme).

By the way, we still have theme cache cleaning issues in v7, that bug is open :(

**Comment by arthurpar06** [comment]
Yeah we need to work on the modules to have `artisan view:cache`, `artisan view:clear` working and also to fix the migrate issue when we activate it from the admin

**Pull request closed without merging**